### ISSUE TITLE
A multiday event that wraps to another row shows a time of 12a on the second row

### ISSUE THREAD

**Issue by JoshMoreno** [fullcalendar/fullcalendar#5826]
### Reduced Test Case
https://codepen.io/jtree5757/pen/OJNwpZO

### Bug Description
A multiday event that wraps to another row shows a time of 12a on the second row.
```
{
    title: 'Conference',
    start: '2020-09-18T13:00:00-07:00',
    end: '2020-09-21T15:00:00-08:00'
}
```

### Screenshots
![2020-09-15 at 10 17 AM](https://user-images.githubusercontent.com/22627952/93243178-b3829780-f73c-11ea-9097-fde2d6ff9850.png)

### Workaround
```js
eventContent: function(args) {
    if (!args.isStart) {
        return args.event.title
    }
},
```

Is this intentional?

Is there a better way of doing multiday events? I was thinking of going the recurring event route but wasn't sure since they're not technically recurring events. For example, a 5-day course is a single event spread over multiple days.

**Comment by acerix** [comment]
I don't believe it's intentional, seems to me it should not show the time on the second row in this case, only at the actual start of the event.  That workaround is a good way to remove it.

In the case of 5-day course though, it may be better to make it an "allDay" event (remove the times from start/end) or you could add each day as it's own event to capture the start/end of each day (better for other views).
